Makefile

```diff
@@ -166,7 +178,7 @@ runserver: ## Run the Django application locally
 	poetry run python ./manage.py runserver 0:8000
 
 .PHONY: dev-init
-dev-init: load-design-system-templates collectstatic compilemessages makemigrations migrate load-topics createsuperuser  ## Run the pre-run setup scripts
+dev-init: load-design-system-templates collectstatic compilemessages makemigrations migrate load-topics createsuperuser npm-install npm-build ## Run the pre-run setup scripts
```

Comment:
Nit: I don't think install should be here, just like poetry install isn't. Happy to keep npm-build. Worth updating readme accordingly around installing dependencies.

Reply:
I agree.

Also, I keep having this niggly feeling that perhaps we're abstracting simple commands in the makefile a bit too much. I get the ones that are unfamiliar perhaps, but isn't `npm install`/`npm run build` better than `make npm-install`/`make npm-build`? Not a blocker for this PR, but something to consider down the line

Reply:
Indeed, our Makefile has become quite large, and we will end up with many aliases for simple commands. I think we should make the README better if things are not clear. I'd encourage devs to customise their own environment by creating local aliases, using `nvm use` at shell startup, and so on. Let's revisit this topic soon.

Reply:
Would the easiest and most consistent solution be to include `npm-install` in the `make install`/`make install-dev` targets? I agree we shouldn't be just abstracting single commands for the sake of it, but bundling them into sensible combinations to make dev workflows simpler seems useful to me.

Reply:
Adam and I had a discussion, and I'll be happy to go with leaving the `npm run build` within `make dev-init`. Happy to add `npm install` under `make install` and `make install-dev`.

Reply:
Done! @MebinAbraham @zerolab @AdamHawtin [ffbbb11](https://github.com/ONSdigital/dis-wagtail/pull/306/commits/ffbbb114d9f7f7615debf84cdb0834d52d9cd552)